Common/Securities/Cash.cs

```diff
@@ -138,6 +138,9 @@ public void EnsureCurrencyDataFeed(SecurityManager securities, SubscriptionManag
                 }
             }
 
+            // get the market from the first existing subscription
+            string market = subscriptions.Subscriptions[0].Market;
```

Comment:
maybe we should be grabbing the market from the first forex subscription? this could be an equity subscription which could cause issues.